CHANGELOG.md

```diff
@@ -116,6 +116,10 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 * New `subject(pattern)` revset function that matches first line of commit
   descriptions.
 
+* The new `jj sign` command allows signing commits.
+
+* The new `jj unsign` command allows unsigning commits.
+
```

Comment:
These do feel a bit scarce, but maybe that's ok? I am open to suggestions of things to add here.